### ISSUE TITLE
Implement RFC7239 support in Proxy and Middleman

### ISSUE THREAD

**Issue by gregw** [jetty/jetty.project#842]
Add configurable support for RFC7239 Forwarded header as an option to the defacto standard X-Forwarded-XXX headers.    This should be for at least the Proxy servlet, but maybe also the Middleman.

**Comment by gregw** [comment]
See also #834 and #185

**Comment by sbordet** [comment]
@gregw Both `ProxyServlet` and `AsyncMiddleManServlet` use `X-Forwarded-XXX` headers from `AbstractProxyServlet.addXForwardedHeaders()`.

Where did you see `ProxyServlet` use RFC 7239 and `AsyncMiddleManServlet` not using those ?

**Comment by joakime** [comment]
Opened new issue specifically for AbstractNCSARequestLog, closing this one.

**Comment by sbordet** [comment]
Reopening, as the mechanism for adding headers in `ProxyServlet` has nothing to do with `AbstractNCSARequestLog`.

**Comment by joakime** [comment]
@sbordet correct, this one should have remained open.   I meant to close #834 instead.

**Comment by sbordet** [comment]
In 12, `ProxyHandler` uses `Forwarded`, while ee10's `ProxyServlet` uses `X-Forwarded-*`.
We should make `ProxyServlet` default on `Forwarded` with an option to use `X-Forwarded-*`.

**Comment by sbordet** [comment]
Due to change in behavior, rescheduled for 12.1.0.

**Comment by sbordet** [comment]
Implemented by #12352.
